apps/framework-cli/src/infrastructure/olap/clickhouse.rs

```diff
@@ -356,58 +355,6 @@ pub async fn check_ready(
     .await
 }
 
-/// Fetches all tables from the ClickHouse database
-///
-/// # Arguments
-/// * `configured_client` - The configured client to use
-///
-/// # Returns
-/// * `Result<Vec<ClickHouseSystemTable>, clickhouse::error::Error>` - List of system tables
-///
-/// # Details
-/// - Excludes system and information schema tables
-/// - Returns table metadata including UUID, name, engine
-/// - Orders results by table name
-///
-/// # Notes
-/// The order of columns in the query must match the order of fields in ClickHouseSystemTableRow
-pub async fn fetch_all_tables(
-    configured_client: &ConfiguredDBClient,
-) -> Result<Vec<ClickHouseSystemTable>, clickhouse::error::Error> {
-    let client = &configured_client.client;
-    let db_name = &configured_client.config.db_name;
-
-    // NOTE: The order of the columns in the query is important and must match the order of your struct fields.
-    let query = "SELECT uuid, database, name, dependencies_table, engine FROM system.tables WHERE (database != 'information_schema') AND (database != 'INFORMATION_SCHEMA') AND (database != 'system')";
-
-    debug!("<DCM> Fetching tables from: {:?}", db_name);
-
-    let tables = client
-        .query(query)
-        .fetch_all::<ClickHouseSystemTableRow>()
-        .await?
-        .into_iter()
-        .map(|row| row.to_table())
-        .collect();
-
-    debug!("<DCM> Fetched tables: {:?}", tables);
-
-    Ok(tables)
-}
-
-/// Fetches tables matching a specific version pattern
```

Comment:
This doc as well please 😄 

I think those are docs I added in other PRs that got merged